### DISCUSSION TITLE
part-ii/2020-134/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#174] [Question comments]
Part II 2020 Riemann Surfaces: Paper 1, Section II, 24F 

https://questions.tripos.org/part-ii/2020-134/

**Comment by silkca** [comment]
The first part is bookwork.

The second part requires you to extend $f$ to $\bar{f}: \mathbb{C_\infty} \rightarrow \mathbb{C_\infty}$, determine the two possible natures of ramification points and branch points of $\bar{f}$, and for each possibility $g$ and $h$ can be explicitly found so that $g \circ \bar{f} \circ h(z)$ restricted to $\mathbb{C}$ is of one of the forms given above. Note the general forms of $g$ and $h$ given above are exactly the general forms of Möbius transformations which fix $\infty$ - this is not of direct importance, but it indicates roughly why $g$ and $h$ can be constructed.
